tools/opensnoop.bt

```diff
@@ -56,6 +54,29 @@ tracepoint:syscalls:sys_enter_openat2
 	@filename[tid] = args.filename;
 }
 
+macro getcwd(@paths) {
+	$dentry = curtask->fs->pwd.dentry;
+
+	$max_path_depth = getopt("depth", 35);
```

Comment:
Did you want to call out in the usage comment above that this is configurable?

Reply:
Yep, thanks!